metricbeat/helper/openmetrics/metric.go

```diff
@@ -250,7 +250,7 @@ func (m *commonMetric) GetValue(metric *prometheus.OpenMetric) interface{} {
 		value := mapstr.M{}
 		if !math.IsNaN(summary.GetSampleSum()) && !math.IsInf(summary.GetSampleSum(), 0) {
 			value["sum"] = summary.GetSampleSum()
-			value["count"] = summary.GetSampleCount()
+			value["count"] = uint64(summary.GetSampleCount())
```

Comment:
Jus for my understanding that this was not needed , just was done for alignment correct?

Reply:
Yes @ishleenk17, that's exactly right. The intent is to keep the same user visible behavior.